suite/src/__tests__/fast/data-feed-api.test.ts

```diff
@@ -0,0 +1,203 @@
+import { describe, test, beforeAll, expect } from '@jest/globals'
+import { newCeramic, waitForAnchor } from '../../utils/ceramicHelpers.js'
+import { createDid } from '../../utils/didHelper.js'
+import { EventAccumulator } from '../../utils/common.js'
+import { StreamID } from '@ceramicnetwork/streamid'
+import { Model } from '@ceramicnetwork/stream-model'
+import { ModelInstanceDocument, ModelInstanceDocumentMetadataArgs } from '@ceramicnetwork/stream-model-instance'
+import { newModel } from '../../models/modelConstants'
+import { CeramicClient } from '@ceramicnetwork/http-client'
+import { CommonTestUtils as TestUtils } from '@ceramicnetwork/common-test-utils'
+import { EventSource } from 'cross-eventsource'
+import { JsonAsString, AggregationDocument } from '@ceramicnetwork/codecs'
+import { decode } from 'codeco'
+
+const ComposeDbUrls = String(process.env.COMPOSEDB_URLS).split(',')
```

Comment:
Just in case, that would result in `["undefined"]` if `COMPOSEDB_URLS` env is empty.

Reply:
Based on the context/env you might want to throw an error if `COMPOSEDB_URLS` can be empty.